queries/angular/highlights.scm

```diff
@@ -101,6 +101,12 @@
   (defer_end_expression)
 ] @punctuation.bracket
 
+(two_way_binding
+  [
+    "[("
+    ")]"
+  ] @punctuation.bracket)
```

Comment:
Is this the only spot that "[(" and ")]" can appear? if so, you don't need to ensure that it is under a `two_way_binding`, you can simply write it like this:

```scm
[ "[(" ")]" ] @punctuation.bracket
```

Reply:
Yes it's only valid within `two_way_binding`

Reply:
If it's only valid under two_way_binding, only the inner part is needed